examples/portable_dashboards_example/public/static_by_reference_example.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import { buildPhraseFilter, Filter } from '@kbn/es-query';
+import type { DataView } from '@kbn/data-views-plugin/public';
+import {
+  LazyDashboardContainerRenderer,
+  DashboardCreationOptions,
+} from '@kbn/dashboard-plugin/public';
+import { EuiCode, EuiPanel, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
+import { ViewMode } from '@kbn/embeddable-plugin/public';
+import { withSuspense } from '@kbn/presentation-util-plugin/public';
+
+const DashboardContainerRenderer = withSuspense(LazyDashboardContainerRenderer);
+
+export const StaticByReferenceExample = ({
+  dashboardId,
+  dataView,
+}: {
+  dashboardId?: string;
+  dataView: DataView;
+}) => {
+  return dashboardId ? (
+    <>
+      <EuiTitle>
+        <h2>Static, by reference example</h2>
+      </EuiTitle>
+      <EuiText>
+        <p>
+          Loads a static, non-editable version of the <EuiCode>[Logs] Web Traffic</EuiCode>{' '}
+          dashboard, excluding any logs with an operating system of <EuiCode>win xip</EuiCode>.
+        </p>
+      </EuiText>
+      <EuiSpacer size="m" />
+      <EuiPanel
+        hasBorder={true}
+        // Once https://github.com/elastic/kibana/pull/145628 is merged, we should (hopefully) be able to make the dashboard
```

Comment:
Is this still true? The PR is merged and working in this version!

Reply:
Oooh, good point! Let me check if I can now contain the entire dashboard in a scrollable container :eyes:

Reply:
Oh man, it works!! That is so cool 🎉 

![Feb-10-2023 12-13-42](https://user-images.githubusercontent.com/8698078/218177913-9c954c3e-90c5-48a9-a4b0-6de5ea5206ec.gif)

I'll merge this - thanks for catching that!

Reply:
Hell yeah! Nice work to you and to @cqliu1! This is great.